travis-ci/init-integration-tests.sh

```diff
@@ -14,22 +14,28 @@ fi
 
 echo "Locking stack for deployment $DEPLOYMENT"
 (
-  set -e
-
   cd example
-
+  set +e
   # Wait for the stack to be available
-  LOCK_EXISTS_STATUS=$(node ./scripts/lock-stack.js true $DEPLOYMENT)
-
+  node ./scripts/lock-stack.js true $DEPLOYMENT
+  LOCK_EXISTS_STATUS=$?
   echo "Locking status $LOCK_EXISTS_STATUS"
 
-  while [ "$LOCK_EXISTS_STATUS" = 1 ]; do
+  while [[ $LOCK_EXISTS_STATUS = 100 ]]; do
+    if [[ $COUNTER -gt 80 ]]; then
```

Comment:
So this is to timeout in Bamboo after 40 minutes? Since we can't rely on travis just crapping out? Maybe we should have the comment if that's the case.

Reply:
Hmm, this should probably just come out for the travis/master PR.      Yes, it's in the 1160 CI fix/branch/etc for that reason.

Reply:
d23c20f68